models/shipping-method/src/shipping-rate/shipping-rate-draft/presets/sample-data-fashion/aud-2000.ts

```diff
@@ -0,0 +1,12 @@
+import { Money } from '@commercetools-test-data/commons';
+import type { TShippingRateDraftBuilder } from '../../../types';
+import * as ShippingRateDraft from '../../index';
+
+const aud2000 = (): TShippingRateDraftBuilder =>
+  ShippingRateDraft.presets
+    .empty()
+    .price(Money.presets.withCent().currencyCode('AUD').centAmount(2000))
+    .freeAbove(Money.presets.withCent().currencyCode('AUD').centAmount(20000))
+    .tiers([]);
```

Comment:
The empty arrays aren't necessary - the API will add them

Reply:
@stephsprinkle In the empty preset I have `tiers` being set to `undefined`, should I remove that?

Reply:
@stephsprinkle In the empty preset I'm setting `tiers` to `undefined` - should I undo that?

Reply:
No, that's totally fine